components/x-follow-button/src/Button.jsx

```diff
@@ -0,0 +1,35 @@
+import { h } from '@financial-times/x-engine';
+import styles from './styles/main.scss';
+
+const Button = ({
+	conceptId,
+	followPlusDigestEmail,
+	isSelected,
+	extraButtonClasses,
+	variant,
+	alternateText,
+	buttonText,
+	name,
+	...props
+}) => (
+	<button
+		data-alternate-text={ isSelected ? `Add ${name} to MyFT` : `Remove ${name} to MyFT` }
+		aria-label={ isSelected ? `Remove ${name} to MyFT` : `Add ${name} to MyFT` }
+		title={ isSelected ? `Remove ${name} to MyFT` : `Add ${name} to MyFT` }
+		data-alternate-label={ isSelected ? `Add ${name} to MyFT` : `Remove ${name} to MyFT` }
+		aria-pressed={ isSelected ? 'true' : 'false' }
+		className={ `${extraButtonClasses ? extraButtonClasses : ''}
+				${styles['n-myft-follow-button']}
+				${variant ? ` n-myft-follow-button--${variant}` : ''}` }
```

Comment:
these classes won't be available in the bundled css output, so you'll need to use ``styles[`n-myft-follow-button--${variant}`]``.

Reply:
this is a style thing, but i'll point out as well that `n-myft-follow-button` is more verbose than needed with x-dash styles. the classnames get randomised when the css is bundled, so something short and descriptive like `styles.button` and ``styles[`button-${variant}`]`` would look a bit nicer and still have no risk of colliding with any other styles